Sources/Application/View/View.swift

```diff
@@ -18,28 +18,52 @@ public class View: Viewable {
 		return Int(maxNumberOfGuesses)!
 	}
 
+	public func promptWordSelectionType() {
+		io.display("Would you like a randomly selected word or phrase, or would you like a friend to select one?")
+		io.display("[1] Random")
+		io.display("[2] Select")
+	}
+
+	public func receiveSelectionType(numberOfOptions: Int) -> Int{
+		var selectionType = io.getUserInput()
+		while (selectionType == "" || !(validSelection(selectionType, numberOfOptions: numberOfOptions))) {
+			io.display("Invalid selection.")
+			selectionType = io.getUserInput()
+		}
+		return Int(selectionType)!
+	}
+
+	public func receiveWord() -> Word {
+		var word = Word(gameWord: io.getUserInput())
```

Comment:
The `View` now has a hard dependency on `Word`. Any thoughts on how you can fix this?